src/Integration.Vsix/SonarLintTelemetryPackage.cs

```diff
@@ -68,6 +69,8 @@ private async System.Threading.Tasks.Task InitOnUIThreadAsync()
                 telemetryManager = await this.GetMefServiceAsync<ITelemetryManager>();
                 logger.WriteLine(Resources.Strings.Telemetry_InitializationComplete);
 
+                await this.GetMefServiceAsync<ICFamilyTelemetryManager>();
```

Comment:
Initializing the event listeners

Reply:
This deserves a comment; it's not obvious why the class is importing a type it isn't using - 
 the new component is completely independent, but we need it to be created somehow, so we're importing it here.